### ISSUE TITLE
Training a smaller model

### ISSUE THREAD

**Issue by ShivanshuPurohit** [openai/vdvae#4]
To reduce the model size for a single GPU, is there anything more to do after reducing the number of residual blocks in `vae.py`?

**Comment by rewonc** [comment]
Sure, you could reduce the `width` parameter (in addition to reducing the number of layers in the decoder or encoder)

**Comment by ShivanshuPurohit** [comment]
Any other tips to train on single V100?

**Comment by rewonc** [comment]
You can just train on a single V100 with the hyperparams as given. You'll
probably have to decrease the learning rate, because the batch size will be
lower.
